javaparser-core/src/main/java/com/github/javaparser/ast/body/ModifierSet.java

```diff
@@ -19,8 +19,11 @@
 package com.github.javaparser.ast.body;
 
 import com.github.javaparser.ast.AccessSpecifier;
+import com.github.javaparser.ast.Modifiers;
+import com.github.javaparser.ast.NodeWithModifiers;
 
 import java.lang.reflect.Modifier;
```

Comment:
In the current nodes it is not treated as a modifier, it is treated differently. In a method you treat it differently (you use the specific methods setDefault/getDefault). I think it makes sense to treat it like this. While in the code you use a keyword to indicate them, in practice it is not a modifier because a method with a body in an interface is a default method and a method without a body is not a default method.